vector-config/src/tchap/res/values/config-features.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm wondering if we need these files in the flavour `tchap` or if we can use the main package instead, will the other variants override the values from the main package?